service/controller/v26/resource/configmap/create.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	"github.com/giantswarm/microerror"
-	apiv1 "k8s.io/api/core/v1"
+	corev1 "k8s.io/api/core/v1"
```

Comment:
why it was `apiv1` is unclear, but set it up to `corev1` as we get used to it.

Reply:
Like Tim said some of the code here is quite old. Thanks for improving it. <3